pySDC/core/Level.py

```diff
@@ -22,6 +22,8 @@ def __init__(self):
         self.updated = False
         self.time = None
         self.sweep = None
+        self.e_embedded = None
+        self.e_extrapolated = None
         # freeze class, no further attributes allowed from this point
```

Comment:
Please be more expressive in your variable names, e.g. `error_est_embedded` or so.

Reply:
Better: be consistent with what you did in the controller, e.g. `error_embedded_estimate`